src/Button.js

```diff
@@ -47,7 +48,7 @@ class Button extends React.Component {
     return (
       <Component
         {...elementProps}
-        type={elementProps.type || 'button'}
```

Comment:
That shouldn't be possible though, since type is defined as `oneOf` the truthy button types above?